src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -491,26 +491,42 @@ public void run() {
                                     }
                                     log.log(Level.INFO, "Found judgement event: " + event);
 
-                                    //process a judgement event
+                                    //process a judgment event
                                     try {
                                         //get a map of the data elements for the judgement
                                         Map<String, Object> judgementEventDataMap = (Map<String, Object>) eventMap.get("data");
-
-                                        // check if this is a "delete" event
                                         String operation = (String) eventMap.get("op");
-                                        if (operation != null && operation.equals("delete")) {
-                                            
-                                            //it is a delete; remove from the global map the judgement whose ID is specified
-                                            String idToDelete = (String) judgementEventDataMap.get("id");
-                                            synchronized (remoteJudgementsMapLock) {
-                                                getRemoteJudgementsMap().remove(idToDelete);
-                                            }
-
-                                            //TODO: how do we notify the local PC2 system that this judgement should be deleted??
```

Comment:
Since DOMJudge does not delete judgements, we do not need this for WF.  Perhaps a separate issue should be created for deleting the judgement, and the comment above moved there?

Reply:
First:  @lane55 :  this section of code is not about deleting RUNS, it is about deleting JUDGMENTS.  

Second, this section of code is not about deleting anything in the PC2 server; it is about deleting a reference to a judgment _in the RemoteEventFeedMonitor's map of remote judgments_ (a static map that maps a _remote system_  judgment id to a record containing the submission id and the judgment type id (i.e. the judgment acronym) separated by a colon (see method `RemoteEventFeedMonitor.getRemoteJudgementsMap()`)).

Third, yes, there is an issue about how to go about informing the PC2 server that a judgment has been deleted.  This was noted in the comment in line 509 in the original code (above). That line has (correctly) been deleted in this PR, and has been (correctly) replaced with a corresponding comment (at line 814) in (new) method `deleteJudgment(eventMap)`, below.     

The issue of how to handle "delete judgment" events from the remote CCS may be important (or maybe not, since as @johnbrvc  points out DOMjudge never sends any such events), but dealing with it is outside the scope of this Issue/PR.

Reply:
understood.